mantisshrimp/backbones/torchvision_backbones.py

```diff
@@ -0,0 +1,85 @@
+# This imports the torchivsion defined backbones
+__all__ = ["create_torchvision_backbone"]
+
+from ..imports import *
+
+
+def create_torchvision_backbone(backbone: str, is_pretrained: bool):
```

Comment:
Can we change `is_pretrained` to `pretrained`?

I see your pointed of having `is_` and it does makes more sense, but I think it's better to follow torchvision convention here, it's what people are already used to

Reply:
Changed to follow convention